deploy/helm/elastic-agent/values.yaml

```diff
@@ -300,6 +300,15 @@ kubernetes:
       # -- system metric stream vars
       # @section -- 2 - Kubernetes integration
       vars: {}
+  event:
+    # -- enable [event](https://www.elastic.co/guide/en/beats/metricbeat/8.11/metricbeat-metricset-kubernetes-event.html)
```

Comment:
let's use this link https://www.elastic.co/docs/reference/integrations/kubernetes/events for that